connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorMakerConfigTest.java

```diff
@@ -49,15 +53,15 @@ public void testClusterConfigProperties() {
             "clusters", "a, b",
             "a.bootstrap.servers", "servers-one",
             "b.bootstrap.servers", "servers-two",
-            "security.protocol", "SASL",
+            "security.protocol", "SSL",
```

Comment:
Why do we need this change?

Reply:
Since validators are added to `security.protocol`, valid values are: PLAINTEXT, SSL, SASL_PLAINTEXT, SASL_SSL.
Make changes here so that testCase can pass normally.

Reply:
Is it possible today to bring up a MirrorMaker 2 instance with `security.protocol` set to `SASL` without it failing? If so, we might have to leave out this change (and the new validator for the `security.protocol` property) to avoid breaking compatibility.